src/main/java/com/easypost/model/Shipment.java

```diff
@@ -35,6 +35,7 @@
 	String uspsZone;
 	Map<String, Object> options;
 	List<ShipmentMessage> messages;
+    List<TaxIdentifier> taxIdentifiers;
```

Comment:
This is indented incorrectly, can we fix it? Also, it should be singular.
```suggestion
    List<TaxIdentifier> taxIdentifier;
```